src/commcare_cloud/ansible/roles/shared_dir/vars/main.yml

```diff
@@ -12,6 +12,7 @@ shared_mount_address: "{{ groups.shared_dir_host[0] }}:{{ shared_data_dir }}"
 restore_payload_dir_host: "{{ shared_data_dir }}/{{ restore_payload_dir_name }}"
 transfer_payload_dir_host: "{{ shared_data_dir }}/{{ transfer_payload_dir_name }}"
 shared_temp_dir_host: "{{ shared_data_dir }}/{{ shared_temp_dir_name }}"
+blobdb_dir_host: "{{ shared_data_dir }}/{{ blobdb_dir_name }}"
```

Comment:
I don't know if a shared directory provides the proper filesystem guarantees needed by the fsdb variant of blob db. It will probably work most of the time, but may have edge cases where things are broken.

Reply:
Thanks for the inputs @millerdev 
I assume you are referring to any new installations that would be done. Any existing setups won't get impacted by this. right?

Reply:
I haven't really considered whether this change would impact existing installations, but I would expect not since we previously did not provide an easy way to put the blob db in a shared directory. And I'm not sure it's a good idea to make it easy since we do not know if it will work. The recommended blobdb configuration for multi-server environments is to either use a third-party S3 service like AWS, or use a self-hosted service like minio.

Reply:
> The recommended blobdb configuration for multi-server environments is to either use a third-party S3 service like AWS, or use a self-hosted service like minio.

I will add that to the recommendations/documents.

The installation currently fails which is not great, so i am going to go ahead with this change unless this is a blocking feedback, then I am happy to reconsider.

Reply:
I've been trying to say I don't think this is a supported configuration, so I don't think it's a good idea to use it for anything other than a test environment. It's hard for me to tell what this PR are geared toward. For example, should these changes only affect test environment setup, or may they also affect production setups? If this could affect production environment setup then I thing we should not make this change.